### ISSUE TITLE
How to change text of dialog box?

### ISSUE THREAD

**Issue by IndustrialOne** [nptr/msstyleEditor#144]
*Windows Version*: <<Windows 10 Pro 21H1 19043.906, ...>>

**Question**
How do I change the text in the dialog box? In the theme I have it is black but it should be white.

**Things you already tried**
I changed all black colors to white but it remained the same.

**Additional context**
https://winclassic.net/thread/2339/help-customizing-theme-text-colors

**Checklist**
- [x] My uxtheme.dll is patched
- [x] I am using the latest version of msstyleEditor
- [x] I checked the Wiki
- [x] None of the other issues cover this

**Comment by e-t-l** [comment]
Hey did you ever figure this out?

**Comment by IndustrialOne** [comment]
Yeah I did.

Personalization > Colors
To restore white text on dialog boxes, select default blue (0,120,215) for accent color and check box for "Title bars and window borders" under "Show accent color on the following surfaces". Turn on transparency effects too just in case.

**Comment by e-t-l** [comment]
> To restore white text on dialog boxes, select default blue (0,120,215) for accent color and check box for "Title bars and window borders" under "Show accent color on the following surfaces".

Ok but that's more of a workaround than an actual answer. It doesn't work for anyone who *doesn't want* title bars and window borders to be default blue (which I imagine is most people)

**Comment by IndustrialOne** [comment]
Yes, it is a workaround because Windows 10 overrides some msstyle settings or blends them with Personalization settings. In other areas the msstyle setting supersedes another setting. You can't control everything with the .msstyles file anymore. Microcock makes it more and more difficult to do any customization. So you gotta find a combination that works. In my case, the default blue in Personalization settings worked for making the title bar text white as it was set in the .msstyles file, anything else made it black.

More investigation is needed for the multiple variables that control colors like these.
